src/controllers/RepostController.ts

```diff
@@ -0,0 +1,142 @@
+                job,
+                "What job would you like to create job posts for?",
+                this.spinner
+            );
+            if (!id) throw new Error(`Job cannot be found with id ${id}.`);
+            jobID = id;
+            this.spinner.start(`Fetching job posts for ${name}.`);
+            jobInfo = await job.getJobData(jobID);
+            if (!jobInfo.posts.length)
+                throw new Error(
+                    `Job posts cannot be found for ${jobInfo.name}.`
+                );
+            this.spinner.succeed();
+            const jobPostID = await getAllJobPostsInteractive(
+                jobInfo.posts,
+                "What job post should be copied?"
+            );
+            const boardToPost = await job.getBoardToPost();
+            const jobPost = new JobPost(page, this.config);
+            const postInfo = jobInfo.posts.find(
+                (post) => post.id === jobPostID
+            );
+            if (!postInfo) throw new Error(`Job post cannot be found.`);
+            await this.createSpecificJobPost(
+                job,
+                jobID,
+                jobInfo,
+                postInfo,
+                boardToPost,
+                page
+            );
+            const deleteSpecificPost = await deleteSpecificPostInteractive(
+                this.config,
+                jobPost,
+                jobInfo,
+                postInfo
+            );
+            if (deleteSpecificPost) {
+                await page.reload();
+                console.log(green("✔"), "Old job post deleted.");
+            }
+        } else {
+            if (!postID)
+                throw new UserError(`Job post ID argument is missing.`);
+            this.spinner.start(`Fetching the job information.`);
+            jobID = await job.getJobIDFromPost(postID);
+            jobInfo = await job.getJobData(jobID);
+            if (!jobInfo.posts.length)
+                throw new Error(
+                    `Job posts cannot be found for ${jobInfo.name}.`
+                );
+            this.spinner.succeed();
+
+            const boardToPost = await job.getBoardToPost();
+
+            const postInfo = jobInfo.posts.find((post) => post.id === postID);
+            if (!postInfo) throw new Error(`Job post cannot be found.`);
+            const jobPost = await this.createSpecificJobPost(
+                job,
+                jobID,
+                jobInfo,
+                postInfo,
+                boardToPost,
+                page
+            );
+            const referrer = joinURL(
+                this.config.greenhouseUrl,
+                `/plans/${jobInfo.id}/jobapp`
+            );
+            await jobPost.deletePost(postInfo, referrer);
+            await page.reload();
+            console.log(green("✔"), "Old job post deleted.");
+        }
+
+        console.log("Happy hiring!");
+
+        browser.close();
+    }
+
+    private async createSpecificJobPost(
```

Comment:
We can simplify it to `createJobPost`, as the word specific doesn't add much meaning. Same for the other method

Reply:
Agree, it was a naming from the initial functionality. Thanks.